### ISSUE TITLE
System is assigned incorrectly in the root element

### ISSUE THREAD

**Issue by jeanetteclark** [NCEAS/metacatui#1371]
**System is automatically filled in as "knb" in the root of the document**

For example, from this [test document](https://test.arcticdata.io/view/urn%3Auuid%3A374a77fd-d350-448f-849e-113d34bcc4f8)

```
<eml:eml xmlns:eml="eml://ecoinformatics.org/eml-2.1.1" 
    xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
    xmlns:stmml="http://www.xml-cml.org/schema/stmml-1.1"
    xsi:schemaLocation="eml://ecoinformatics.org/eml-2.1.1 eml.xsd"
    packageId="urn:uuid:374a77fd-d350-448f-849e-113d34bcc4f8"
    system="knb">
```

**To Reproduce**
 1. Create a test dataset on test.arcticdata.io
 2. Examine root element

**Expected behavior**
I believe that the system should be the URL of the member node (eg: https://arcticdata.io, https://knb.ecoinformatics.org)

**Desktop (please complete the following information):**
 - OS: Mojave 10.14.4 
 - Browser: Chrome
 - Version: 80.0.3987.163 (Official Build) (64-bit)

**Comment by amoeba** [comment]
Can confirm a new document has a `system` of "knb" on test.arctic. This is controlled by `EML211.createXML()` 

```js
createXML: function() {
  ...
  eml.attr("system", "knb"); // We could make this configurable at some point
  ...
```

See: https://github.com/NCEAS/metacatui/blob/master/src/js/models/metadata/eml211/EML211.js#L1903

Can also confirm the editor doesn't mess with `system` when it's set on a document we're updating so, depending on what's changed here, we may want to tweak that too.

It strikes me that we haven't been consistent on how we set `system` in the past, `system` is a bit of a historical vestige, and I'm not sure if we have a standard approach right now. @laurenwalker, do you remember if we have one here?

**Comment by laurenwalker** [comment]
No, I'm not aware of a standard approach. According to the EML spec, a URL to the data management system is suggested: https://eml.ecoinformatics.org/schema/eml-resource_xsd.html#SystemType

So I think we could use the repository base URL. (We could use the AppModel `baseUrl` attribute)

**Comment by mbjones** [comment]
"knb" has been our historical `system` since when the 'Network' in the KNB was the emphasis (i.e., as the precursor to DataONE). Its also why all of hte LTER identifiers have 'knb' in them. We've recognized for a long time that a bare 'knb' word is insufficient to scope things well, and that a URI would be better.  There's no agreed upon vocabulary for system designators, and in reality the `system` attribute has little meaning.  @jeanetteclark and I discussed changing it to something a bit more meaningful, either a DataONE URI for dataone-scoped identifiers (e.g., in which resolve() would work), or possibly as a DOI uri where that resolve service would work.

**Comment by amoeba** [comment]
I think I'd put some votes (if I get multiple) for any of

1. @laurenwalker's repo-specific `system` value based on `baseUrl`
2. 
  ```xml
  <eml packageId="foo.1.1" system="dataone" ... />...</eml>
  ```
3. 
  ```xml
  <eml packageId="foo.1.1" system="dataone.org" ... />...</eml>
  ```
4. 
  ```xml
  <eml packageId="foo.1.1" system="https://dataone.org/datasets/" ... />...</eml>
  ```

**Comment by mbjones** [comment]
I like number 4 as it both provides a URI that leads to dataset collections, and combined with the identifier should create a resolvable URI. For Metacat systems that are not (and will not be( connected to DataONE as members, then number 1 would make sense.
